docs/Plugins.md

```diff
@@ -1,36 +1,39 @@
 Plugins
 =======
 
-Moya plugins are used to perform some side effect whenever a request is sent or received. Plugins are defined by the `PluginType` protocol, which can receive callbacks when:
-- a request is going to be sent;
+Moya plugins are used modify requests and responses or perform side-effects. A
+plugin is called when:
+- a request is being prepared.
```

Comment:
Maybe we can be more descriptive about events and in the parenthesis add the function name? So we can describe it freely and still have reference in the line. What do you think?

Also I've just noticed that you missed "to" in:
> plugins are used **to** modify requests